test/index.test.ts

```diff
@@ -4,51 +4,132 @@ import expect from 'expect';
 
 import { WildlinkClient } from '../src';
 
-describe('WildlinkClient', () => {
-  const WLClient = new WildlinkClient('SECRET');
-  describe('getDeviceToken()', () => {
-    it('should be defined', () => {
-      expect(WLClient.getDeviceToken).toBeInstanceOf(Function);
+const SECRET = 'SECRET';
+const DEVICE_ID = 0;
+const DEVICE_TOKEN = 'DEVICE_TOKEN';
+const DEVICE_KEY = 'DEVICE_KEY';
+const DEVICE = {
+  DeviceID: DEVICE_ID,
+  DeviceToken: DEVICE_TOKEN,
+  DeviceKey: DEVICE_KEY,
+};
+
+describe('WildlinkClient', (): void => {
+  describe('constructor', (): void => {
+    it('should throw if no secret', (): void => {
+      try {
+        new WildlinkClient('', 1);
+        expect((): void => {}).toBeCalledTimes(0);
```

Comment:
looks like an unnecessary check

Reply:
will it even reach this step?

Reply:
it will not thats why it is expected to call 0 times. this is meant to catch if the constructor doesnt throw